README.md

```diff
@@ -1 +1,163 @@
-# Group-Recipe-app
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+<!-- - [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq) -->
+- [📝 License](#license)
+
+# 📖 Recipe App <a name="about-project"></a>
+
+This project a functional Recipe app created using Ruby on Rails. It allows user to create multiple recipes, foods, inventory lists and connect them together.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
+
+  <ul>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+
+### Key Features <a name="key-features"></a>
+
+- **Recipe**
+- **Authentication and Authorization**
+- **Table Logic**
+- **Add Data to DB**
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- IDE or code editor: **VsCode, atom, sublime**...
+- To have [Ruby](https://www.ruby-lang.org/en/) and [Git](https://git-scm.com/) installed on your system.
+- To have [Node.js](https://nodejs.org/) installed on your system.
+
+After installing Ruby, run the following command to install rails.-
+
+```sh
+ gem install rails
+```
+
+### Setup
```

Comment:
- [ ] I am unable to open the app on the live server by using the instructions provided by you. Kindly provide proper instructions.

Reply:
Following commit for the solution:
https://github.com/Linushaddai99/Group-Recipe-app/pull/33/commits/44765d251db1f63cfb5b477f68e8a063c0a1d791